src/renderingHtml.cpp

```diff
@@ -22,6 +22,8 @@
 #include "global.h"
 #include "stringTools.h"
 
+std::string htmlcss;
+
```

Comment:
This instantiation would better be in req.cpp with the other "static objects".

Reply:
OK